content/en/docs/tasks/observability/kiali/index.md

```diff
@@ -285,6 +285,59 @@ Force an invalid configuration of a service port name to see how Kiali reports a
 
     {{< image width="80%" link="./kiali-validate3-ok.png" caption="Service Details Showing Valid Configuration" >}}
 
+## Viewing and editing Istio configuration YAML
+
+Kiali provides a YAML editor for viewing and editing Istio configuration resources. The YAML editor will also provide validation messages
+when it detects incorrect configurations.
+
+1.  Create Bookinfo destination rules:
+
+    {{< text bash >}}
+    $ kubectl apply -f samples/bookinfo/networking/destination-rule-all.yaml
+    {{< /text >}}
+
+1.  Click `Istio Config` on the left hand navigation bar to navigate to the Istio configuration list.
```

Comment:
```suggestion
1.  Click **Istio Config** on the left hand navigation bar to navigate to the Istio configuration list.
```

Reply:
Take a look at the commit message I wrote for this very change:

```
The linter is failing this because it considered "Config" a spelling error.
However, the actual GUI menu item is literally called "Istio Config".
Because of this, I cannot format "Istio Config" in simply bold letters
(which is the Istio doc standard for denoting GUI elements).
Thus, even though its a GUI element, I have to surround with
backticks to avoid this being considered a spelling error.
```

Reply:
@geeknoid what is the workaround other than removing the double-astericks (bold) and replacing with backticks?

Reply:
I don't have a workaround, just go with the backticks.